### ISSUE TITLE
All Images Have Failed to Resize... (center_crop)

### ISSUE THREAD

**Issue by Shiran-Yuan** [rom1504/img2dataset#441]
**TLDR: Used center_crop, code did not center-crop, why??**

Thank you very much for the awesome work! You're making my life a lot easier.

However, I did meet some problems. I was trying to download a manually filtered subset of LAION. Specifically, all images in the parquets (the `url_list` below) have both width and height larger than 512. So I ran this:

`img2dataset --url_list parquets/test/ --input_format "parquet" --url_col "URL" --output_format files --output_folder hires6 --processes_count 16 --thread_count 64 --image_size 512 --resize_only_if_bigger=True --resize_mode="center_crop" --skip_reencode=True`

Key part: `--image_size 512 --resize_only_if_bigger=True --resize_mode="center_crop"`

But the images were not resized! I looked at the terminal and got _this_:

`total   - success: 0.007 - failed to download: 0.242 - failed to resize: 0.751 - images per sec: 208 - count: 30192`

In other words, almost all successfully downloaded images were not resized. However, it all works perfectly when I replace `center_crop` with `border`.

I'm wondering what may the problem be, especially since I'm not seeing this problem already in the issues (very surprisingly). By the way, I would also like to thank you for having actively responded issues :-)

I am looking forward to your helpful reply, and thank you so much in advance.

**Comment by Shiran-Yuan** [comment]
Refencing #433, I deduced albumentations might be the troublemaker. I downgraded it to 0.5.2 (the last version before 1.0.0), and it solved my problem. 

Suggested fix: alter the default version of albumentations in `requirements.txt`.

**Comment by Happyprince65** [comment]
Just for resizing all images in a folder you can use https://github.com/Happyprince65/Resize-all-Images-in-a-folder-nft-traits
node js code

**Comment by rom1504** [comment]
Running resizing after download is way too costly
